### PR TITLE
feature (#58): Enable spec running for Vue + TS files ; add base specs for each component

### PR DESCRIPTION
**Pull request by jackkoppa**
😀

Addresses #58

(re-opening #59 after that PR seemed to not update w/ all the commits on this branch)

### CODE REVIEW

**Comment by jackkoppa** [comment]
hoping to see check fail for spec w/ intentional failure 🤞

**Comment by jackkoppa** [comment]
Whooo! Fails intentionally & correctly w/ GitHub Action!

Screen Shot 2020-12-16 at 4 10 38 AM

**Comment by jackkoppa** [comment]
Now pushing spec fix...

File: specs/SelectMulti.spec.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import { mount, Wrapper } from '@vue/test-utils'
+
+import SelectMulti from '../src/SelectMulti.vue'
```
[the remark this answers is not in the document]
Comment (jackkoppa) [reply specs/SelectMulti.spec.ts:3]: Yeah, absolutely - that's a great point. Having the linting package eventually setup will be one big help, but the other will be a `CONTRIBUTING.md` file, which GH integration pulls in at the relevant points during a PR process.

Adding that as a task here

**Review by RED143 — APPROVED** [review]
Looks good for me

**Comment by jackkoppa** [comment]
Merging

**Pull request merged**